test/test_util/test_data.py

```diff
@@ -347,7 +625,7 @@ def test_reduce_dataset_subsampling_explicit_values(memory_limit, precision, tas
             memory_limit=memory_limit,
             is_classification=task in CLASSIFICATION_TASKS,
             operations=['precision', 'subsample'],
-            multiplier=8
+            memory_allocation=0.1
```

Comment:
Didn't you change this back to ten in the other PR?

Reply:
I thought I did but apparently it was only the default parameter. `0.1` is equivalent to `10` as it was before. The float value is a bit more intuitive than an integer multiplier.

The values in the fixture dict are more similar to what they were before this whole dataset reduction saga which is a good sign.